scout/server/blueprints/public/views.py

```diff
@@ -27,14 +27,7 @@ def index():
     if current_app.config.get("LDAP_HOST"):
         form = LDAPLoginForm()
 
-    badge_name = current_app.config.get("ACCREDITATION_BADGE")
-    if badge_name and not Path(public_bp.static_folder, badge_name).is_file():
```

Comment:
Removing this code, since in the worse case scenario the image won't show on the index file, no biggie!